neonvm/controllers/alerttracker/tracker.go

```diff
@@ -0,0 +1,87 @@
+package alerttracker
+
+import "time"
+
+var Now = time.Now
+
+// Tracker is a struct that keeps track of failure/success events for a given key.
+// Failure state over a given interval will trigger a firing alert.
```

Comment:
Could you expand on the expected semantics here?

Reply:
I'd love to, but I think we should first settle on better "legend" for what it does, so the fact that those alerts are not grafana alerts doesn't need to be explained in the comments.